lib/plugins/chat.js

```diff
@@ -182,7 +182,15 @@ function inject (bot, options) {
       lookedAtBlock: position
     })
 
-    const [packet] = await once(bot._client, 'tab_complete')
+    // Doing promise race, because of 'tab_complete' event never happening,
+    // in situation when there was no completions for tabbed command.
+    const eventPromise = once(bot._client, 'tab_complete')
```

Comment:
We already have an onceWithCleanup function that handles timeouts. You should use that instead here.

Reply:
I think now is good?